### ISSUE TITLE
[BUG] Manual preheat for PC not going over 170C

### ISSUE THREAD

**Issue by m-cas** [prusa3d/Prusa-Firmware-Buddy#3708]
Please, before you create a new bug report, please make sure you searched in open and closed issues and couldn't find anything that matches.

**Printer type** - [MINI, MK4, XL]
MK4

**Printer firmware version** - [e.g. 4.0.5, ...]
5.1.2

**Original or Custom firmware** - [Original/Custom]
Original - kit

**Optional upgrades** - [e.g. Filament Runout Sensor]
prusa enclosure, silicon socket

**USB drive or USB/Octoprint**
USB flash drive 

**Describe the bug**
Previous print with PC blend finished. I wanted to raise the temp to clean the nozzle, so I selected "preheat -> PC"; the target temp (250) is shown but the actual temp never exceeds 170C; reset didn't work in making it go above 170. I tried with "change filament" and the temp went above 170.

**How to reproduce**

**Expected behavior**
 Achieve the 250C

**G-code**

**Crash dump file**

**Video**

**Comment by murk-sy** [comment]
This is intended behavior, see #3074 
Its main purpose is to preheat for printing (temperatures intended to start bed leveling in a print)

**Comment by m-cas** [comment]
sorry I allow myself to disagree :D  If (and the linked issue has the same problem) the purpose it to raise the temp to 170 (that is the temp for the bed routines) why on hearth it shows the various materials (pets, pc, etc) ? It is confusing and it takes up space on the menu and code for nothing; it should just read "preheat the bed for start of print operations"!!!

**Comment by murk-sy** [comment]
Well I agree in fact, and made a feature request in regards to this, see #1795 
If you've got any ideas for improving it further, I'd appreciate sharing your thoughts there

**Comment by m-cas** [comment]
I think it should go in one of two possible directions:
1. leave the materials list and allow the temps to go up to the material's limit BUT add one entry for "only warm the table for print operations"
2. drop the whole menu and add one entry somewhere that say "warm the table for print operations"

As it stands at the moment it doesn't make any sense as it allows a choice that is limited at 170 for all entries - waste of space and confusing like crazy.

-

**Comment by DoubleStrike** [comment]
Yeah - this is definitely a UI/UX bug.  Users should not have to read reference documentation to understand what a simple option on a screen does.  If preheat only goes to 170, it should clearly state that on the screen - there is literally no point listing 230 or other temperatures if the preheat does not achieve them.

At best it is redundant and ugly UI; at worst, it is inaccurate and confusing.  I disagree with closing the linked bug, as it is clearly a UI issue that needs resolution.  Simply saying "it is bad design as intended" is not an acceptable answer.

**Comment by bkerler** [comment]
Not a bug, but a feature request, but also a necessary one as it is very confusing as it is right now.

**Comment by bkerler** [comment]
I wrote a PR #3709 to fix this confusing ui issue, now showing the right temperature in the preheat menu.

**Comment by m-cas** [comment]
@bkerler Bjoern, I very respectfully disagree, if a menu is telling me that the temp will go up to 250C (target temp on the bottom of the display in red) and instead it never passes 170C, it is not a new feature to have this fixed, it is a bug 😄 
And whoever closed the previous bug with "work as planned" was -again very respectfully- plainly wrong. Let's be honest, the GUI for this "warm up the machine" part is simply broken, mistakes happens and nobody is suing prusa for this, but this is a bug.

**Comment by bkerler** [comment]
Sure, I do understand your confusion as the ui clearly displays a higher temp than expected. But that is due to the fact that the filament selection doesn't show the preheat temp normally but just the nozzle temp, which isn't a bug but intentional for filament change menu.

Of course, the filament display shouldn't have happened here but should have instead displayed its own menu (which refer to as feature request as a new feature had to be developed).

But if you select preheating, it means preheating and not heating up to the target temperature.

Anyways, PR is written and the PR will fix this issue hopefully ;)
